tests/Agent/IntegrationTests/ContainerIntegrationTests/Tests/LinuxContainerTests.cs

```diff
@@ -42,9 +58,20 @@ protected LinuxContainerTest(T fixture, ITestOutputHelper output) : base(fixture
     [Fact]
     public void Test()
     {
-        var actualMetrics = _fixture.AgentLog.GetMetrics();
+        var actualMetrics = _fixture.AgentLog.GetMetrics().ToList();
 
         Assert.Contains(actualMetrics, m => m.MetricSpec.Name.Equals("WebTransaction/MVC/WeatherForecast/Get"));
+
+        // verify 8T metrics
+        var expectedSeenCount = 4;
+
+        var expectedMetrics = new List<Assertions.ExpectedMetric>
+        {
+            new Assertions.ExpectedMetric { metricName = @"Supportability/InfiniteTracing/Span/Seen", callCount = expectedSeenCount },
+            new Assertions.ExpectedMetric { metricName = @"Supportability/InfiniteTracing/Span/Sent", callCount = ExpectedSentCount },
+        };
+
```

Comment:
Should these new 8T tests be a separate set of tests instead of using 8T as our default set of tests for Linux?

Reply:
I'm not sure it makes much difference either way - I would kind of expect to see the container integration tests evolve over time to include specific things we want to test on Linux, and adding separate tests for every one of them would explode the number of containers that have to spin up and run.

Reply:
(maybe I wasn't clear, but for tests we want to run across all distros, it seems to make more sense to put them all in a single test class rather than separate - that would lead to the explosion of containers if we have multiple tests running on all distros...)

Reply:
My main concern about including 8T as the default in these tests, is that most customers do not use 8T, and it results in us not testing normal span events.

Reply:
I can certainly split these out to separate tests, but it's going to essentially double the runtime for the container tests.

Reply:
Given that 8T is a major feature, it might be good to test more thoroughly and often.